lib/src/base-styles/_nx-global-header.scss

```diff
@@ -19,7 +19,10 @@
 
   .nx-back-button {
     @include nx-text-helpers.truncate-ellipsis;
-    margin: 0 var(--nx-spacing-8x) 0 0;
+    // The negative margins and added paddings are added to
+    // fix the blue outline being hidden by overflow: hidden when focused.
+    margin: -4px var(--nx-spacing-8x) -4px -4px;
```

Comment:
The top and bottom margins here aren't necessary.  The element has plenty of extra room in its parent and is vertically centered via flexbox properties